web/template/unlink_person.gohtml

```diff
@@ -0,0 +1,96 @@
+{{ template "page" . }}
+
+{{ define "title" }}Unlink Record{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            {{ if .Success }}
+                {{ template "success-banner" "You have successfully unlinked the record(s)." }}
+            {{ end }}
+
+            <h1 class="govuk-heading-l app-!-embedded-hide">Unlink Record</h1>
+
+                <table class="govuk-table">
+                    <caption class="govuk-table__caption govuk-table__caption--m">Primary record</caption>
+                    <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Name</th>
+                        <td class="govuk-table__cell">{{ .Person.Firstname }} {{ .Person.Surname }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Contact ref #</th>
+                        <td class="govuk-table__cell">{{ .Person.UID }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Address</th>
+                        <td class="govuk-table__cell">{{ .Person.AddressLine1 }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Date of birth</th>
+                        <td class="govuk-table__cell">{{ .Person.DateOfBirth }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Number of links</th>
+                        <td class="govuk-table__cell">{{ len .Person.Children }}</td>
+                    </tr>
+                    </tbody>
+                </table>
+
+            <form class="form" method="POST">
+                <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"/>
+
+                <p class="govuk-heading-m">Secondary records</p>
```

Comment:
I wonder if it would be nice to show a message and no buttons if there are no child records, kind of like in `delete_relationship.gohtml`

Reply:
yeah thats a good idea